testrunner.py

```diff
@@ -58,24 +86,30 @@ def main():
 def test(path, showall: bool) -> bool:
     sys.stdout.write(f'Testing {path}...')
     sys.stdout.flush()
-    
+
     result = None
 
-    with open(path) as f:
-        args = [FRONTEND_BINARY, path]
-        args.extend(sys.argv[1:])
-        try:
-            result = subprocess.run(
-                    args,
-                    encoding='utf-8',
-                    stdout=subprocess.PIPE,
-                    stderr=subprocess.PIPE,
-                    timeout=60)
-        except subprocess.TimeoutExpired:
-            pass
-    
-    filename = os.path.basename(path)
-   
+    if not isinstance(path, list):
```

Comment:
Coudln't we just have path always be a list, and have it contain a single path for non-multi-module tests?

Reply:
Done!